archiver/direct.py

```diff
@@ -9,6 +9,7 @@
 from submission_broker.services.biosamples import BioSamples, AapClient
 from submission_broker.services.biostudies import BioStudies
 
+from submitter.biosamples_submitter_service import BioSamplesSubmitterService
 from submitter.biostudies import BioStudiesSubmitter
 from converter.biostudies import BioStudiesConverter
 from submitter.biostudies_submitter_service import BioStudiesSubmitterService
```

Comment:
Should `BioStudiesSubmitterService` be in the same package as `BioStudiesConverter`?

Reply:
What would be the reason to put them into the same package?
They are doing different things: converters are converting between different formats, submitters are interacting with the various archives and submitting data into them.

Reply:
Sorry, I meant to ask whether `BioStudiesSubmitterService` belongs together with `BioStudiesSubmitter`. 
The name `submitter` repeats in `submitter.biostudies_submitter_service`